micrometer-jakarta9/src/test/java/io/micrometer/jakarta9/instrument/mail/DefaultMailSendObservationConventionTests.java

```diff
@@ -80,34 +80,20 @@ void shouldHaveLowCardinalityKeyValues() throws MessagingException {
         message.setSubject("Test Subject");
 
         MailSendObservationContext context = new MailSendObservationContext(message, "smtp", "example.com", 587);
-        assertThat(convention.getLowCardinalityKeyValues(context)).contains(
-                KeyValue.of("server.address", "example.com"), KeyValue.of("server.port", "587"),
-                KeyValue.of("network.protocol.name", "smtp"));
-    }
-
-    @Test
-    void shouldHandleMessagingException() {
```

Comment:
I dropped this as it seems to be a duplicate of the `subjectShouldBeUnknownWhenFetchingFails()`.